public/templates/public/staff_public_page.html

```diff
@@ -38,8 +38,18 @@ <h4>Contact</h4>
     <p>
       {% if email %}
         <a href="mailto:{{ email|urlencode }}">{{ email }}</a><br>
-      {% endif %}
-      {% staff_faculty_exchanges_phone_numbers staff_page %}
+	{% endif %}
+	{% staff_faculty_exchanges_phone_numbers staff_page %}
+	{% if libcal_id %}
+	  <script src="//ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	    <script>
+	     jQuery.getScript("https://api3.libcal.com/js/myscheduler.min.js", function() {
+		 var id_num = "{{ libcal_id }}";
+		 jQuery("#mysched_{{ libcal_id }}").LibCalMySched({iid: 482, lid: 0, gid: 0, uid: id_num, width: 560, height: 680, title: 'Make an Appointment', domain: 'https://api3.libcal.com'});
+	     });
```

Comment:
There hasn't been a way around this for any Springshare widgets. For the hours tables we have to list each script individually since each one connects to the specific id of that button or table or whatever we are calling.